src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -60,6 +61,9 @@
 	private static final String JAVA_BYTE = "java.lang.Byte";
 	private static final String JAVA_SHORT = "java.lang.Short";
 
+	private static final String JAVADOC_LINK_START = "{@link";
+	private static final String JAVADOC_LINK_STOP = "}";
```

Comment:
Hey Martijn,

`{@link .. }` is niet de enige annotatie. Zo heb je ook

- `{@code .. }`
- `{@docRoot .. }`
- `{@inheritDoc .. }`
- `{@linkplain .. }`
- `{@literal .. }`
- `{@value .. }`

Reply:
Ik test nu niet meer op "{@link", maar op "{@". Ik heb niet getest of ik nu te veel JavaDoc tekst plat sla. Laat maar weten als het slimmer moet.